CrazyCanvas/Saved_Servers.json

```diff
@@ -1,3 +1,8 @@
 {
-    "SERVERS": []
+    "SERVERS": [
+        {
+            "Name": "Chris's server",
+            "Address": "81.170.143.133:4444"
+        }
```

Comment:
This will not cause any issues any more?

Reply:
As long as people do use a branch based on master later than 24/11/2020